region/aggregate-regions.yaml

```
@@ -15,13 +15,29 @@ Europe (excl. Turkey):
 Europe (incl. Turkey):
    definition: 'EU27 + EFTA + AL + BA + BY + ME + MK + UA + UK + RS + TR + XK'
 
+Pan-Europe:
+   definition: 'EU27 (excl. Malta & Cyprus) +CH + BA + ME + RS + MK + AL + NO + UK'
+   source: definition of the pan-european electricity system used in e-highways2050 and plan4res
+   
 # add more defintion of `Europe (*)` above this line
 
 EFTA:
    definition: 'IS + LI + NO + CH'
    notes: 'European Free Trade Association'
    source: 'https://en.wikipedia.org/wiki/European_Free_Trade_Association'
 
+BALKAN:
+   definition: 'SI+RO+HR+BA+ME+RS+BG+MK+GR+AL'
+   source: plan4res public dataset (D4.5)
+   
+SCAND:
+   definition: 'DK+FI+NO+SE'
+   source: plan4res public dataset (D4.5)
+   
+BALTIC:
+   definition: 'EE+LT+LV'
```

same rationale as above

```suggestion
Baltics:
   definition: 'EE + LT + LV'
```